fraud-detection-update-sample/fraud_detection_process_sample.py

```diff
@@ -8,6 +8,9 @@
 """Global Variable definition"""
 
 ISOSCAPES_EE_PATH = 'projects/kazob-370920/assets/isoscapes'
+_ENABLE_d18O_ANALYSIS = True
```

Comment:
Please put comments on these to make it clear when and why you would turn them to true.